plugins/LilyPad/LilyPad.rc

```diff
@@ -450,8 +450,8 @@ BEGIN
     CONTROL         "Raw input",IDC_KB_RAW,"Button",BS_AUTORADIOBUTTON,22,40,112,10
     CONTROL         "DirectInput",IDC_KB_DI,"Button",BS_AUTORADIOBUTTON,22,52,112,10
     GROUPBOX        "Game Device APIs",IDC_STATIC,16,70,191,62
-    CONTROL         "DirectInput",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,65,10
-    CONTROL         "XInput",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
+    CONTROL         "DirectInput (For older controllers. Disable for DS4.)",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,183,10
+    CONTROL         "XInput (For modern controllers.)",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
```

Comment:
The extra info should be in the new tooltips. You can do `DirectInput (Legacy)`.

Reply:
The tooltips for DirectInput and others aren't working, that's the reason i did it this way.

Reply:
You're missing a line for the tooltip.
https://github.com/PCSX2/pcsx2/blob/9aab6ae7fad4d34b5888404c6d4a5bc32496e307/plugins/LilyPad/Config.cpp#L2346

Reply:
Thanks, i'll make the changes.

Reply:
![image](https://user-images.githubusercontent.com/24227051/91754362-92ad3480-ebc9-11ea-8efb-5070eeaf2c03.png)
Works now, i'll revise the text.